skimage/segmentation/active_contour_model.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+from skimage import img_as_float
+import scipy.linalg
+from scipy.interpolate import RectBivariateSpline
+from skimage.filters import sobel
+
+def active_contour_model(image, snake, alpha=0.01, beta=0.1,
+                         w_line=0, w_edge=1, gamma=0.01,
+                         bc='periodic', max_px_move=1.0,
+                         max_iterations=2500, convergence=0.1):
+    """Active contour model.
+
+    Active contours by fitting snakes to features of images. Supports single
+    and multichannel 2D images. Snakes can be periodic (for segmentation) or
+    have fixed and/or free ends.
+
+    Parameters
+    ----------
+    image: (N, M) or (N, M, 3) ndarray
```

Comment:
add space before colon (that's the Numpy docstring standard), here and for the other parameters